fluent-dom/src/dom_localization.js

```diff
@@ -168,6 +168,9 @@ export default class DOMLocalization extends Localization {
 
     if (this.roots.size === 0) {
       this.mutationObserver = null;
+      if (this.windowElement) {
+        this.windowElement.cancelAnimationFrame(this.pendingrAF);
+      }
```

Comment:
For clarity, should also check if we actually have a pending animation frame.

```suggestion
      if (this.windowElement && this.pendingrAF) {
        this.windowElement.cancelAnimationFrame(this.pendingrAF);
      }
```

Reply:
Fixed now, thanks for the review.